src/main/java/se/kth/sda/skeleton/posts/exeption/ResourceNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package se.kth.sda.skeleton.posts.exeption;
```

Comment:
- should be **ResourceNotFoundException** ( grammatic missing a "c" )
- request to move this class to ...skeleton/exception/ResourceNotFoundException.java